stylus/ui-components/modals.styl

```diff
@@ -52,17 +52,23 @@ $modals
         // hide the border-radius
         overflow          hidden
 
-        .coz-modal-content,
-        .coz-modal-section
+        .coz-modal-content
             padding 0 1.5em
 
         .coz-modal-section
+            padding 1.5em
+
+        .coz-modal-section
             border-top 1px solid grey-09
 
+        .coz-modal-content:first-of-type
```

Comment:
`:first-of-type` is dangerous ground. This is not really maintenance friendly.
I don't know what you're reffering to but it seems to me that better class architecture would be a better option.
I get that the use of `:first-of-type` was here before but it gets more and more messy which just prove that maintenance will be difficult.